docs/docs/channels/sms/sns.md

```diff
@@ -0,0 +1,25 @@
+# AWS SNS
+
+You can use the [AWS SNS](https://aws.amazon.com/sns/) provider to send transactional emails to your customers using the Novu Platform with a single API to create multi-channel experiences.
+
+## Getting Started
+
+To integrate AWS SNS on the Novu platform., you need IAM user who has access to `sns:Publish` Action.
+
+## Create User
+
+To create user, login to AWS Console and follow this steps,
```

Comment:
Update with "these steps:"

Reply:
Thanks @florian-lefebvre for review, Will take extra care next time.